### ISSUE TITLE
Linux Support

### ISSUE THREAD

**Issue by billbeans** [AlirezaKJ/BetterSoundCloud#28]
Can this work on Linux? I followed the instructions to build myself with npm and when I run `npm start` I get this:

```
Error: ENOENT: no such file or directory, mkdir 'C://BetterSoundCloud'
    at Object.mkdirSync (node:fs:1372:26)
    at Object.<anonymous> (/home/anon/source-code/BetterSoundCloud/main.js:30:8)
    at Module._compile (node:internal/modules/cjs/loader:1373:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1432:10)
    at Module.load (node:internal/modules/cjs/loader:1215:32)
    at Module._load (node:internal/modules/cjs/loader:1031:12)
    at c._load (node:electron/js2c/node_init:2:17025)
    at cjsLoader (node:internal/modules/esm/translators:352:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:297:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
A JavaScript error occurred in the main process
```

I think the `mkdir 'C://BetterSoundCloud` bit is the problem here, but I'm not a developer. On Linux there is no C: drive

**Comment by AlirezaKJ** [comment]
yeah thats the main issue on linux, im doing a complete rework on the project and it is based on "electron forge" and "app directory variable" which make it fine for linux. the rework will be released soon enough, i will attach the release link it as soon as it work out

**Comment by billbeans** [comment]
I would really appreciate it if you came out with a Linux release. There are currently no desktop clients for SoundCloud on Linux, ever since the API change.

**Comment by Krafting** [comment]
Linux Support would be nice to have yeah !

And then publishing as a Flatpak (Packaging format on Linux) on Flathub (A big "App Store" on Linux) would be even better (If you need help creating a flatpak once linux support has landed, I'll be happy to help ! 😄  )

**Comment by billbeans** [comment]
I've been using [soundcloud-rpc](https://github.com/richardhbtz/soundcloud-rpc) for now, which is basically just an Electron webapp, but it works. BetterSoundCloud looks a lot more interesting, so I'm still holding out for a Linux release!

**Comment by AlirezaKJ** [comment]
its really close, i will be releasing linux packages alongside it.
